packages/sycamore/src/utils.rs

```diff
@@ -40,3 +46,135 @@ pub fn initial_node<G: GenericNode>(_el: &G) -> Option<View<G>> {
         None
     }
 }
+
+#[doc(hidden)]
+pub fn erase_handler<'a, Ev, Handler, S>(
+    cx: Scope<'a>,
+    mut handler: Handler,
+) -> Box<dyn FnMut(JsValue) + 'static>
+where
+    Handler: EventHandler<'a, JsValue, Ev, S> + 'a,
+    Ev: EventDescriptor<JsValue>,
+{
+    let boxed: Box<dyn FnMut(JsValue)> = Box::new(move |e: JsValue| handler.call(cx, e.into()));
+    unsafe { mem::transmute(boxed) }
+}
+
+/// Apply an `AttributeValue` to an element. Used by the `view!` macro.s
+pub fn apply_attribute<'cx, G: GenericNode<AnyEventData = JsValue, PropertyType = JsValue>>(
+    cx: Scope<'cx>,
+    el: G,
+    name: Cow<'static, str>,
+    value: AttributeValue<'cx, G>,
+) {
+    match value {
+        AttributeValue::Str(s) => {
+            el.set_attribute(name.clone(), Cow::Borrowed(s));
+        }
+        AttributeValue::DynamicStr(mut s) => {
+            create_effect(cx, {
+                let name = name.clone();
+                move || el.set_attribute(name.clone(), Cow::Owned(s()))
+            });
+        }
+        AttributeValue::Bool(value) => {
+            let stringified = match value {
+                true => "true",
+                false => "false",
+            };
```

Comment:
If that's the case then your `view!` macro code is also wrong, it's copy pasted from that. Should I change it in both places or leave it for now?

Reply:
Since it's a different situation (`view!` returns an entry in the static attributes, this util doesn't) I changed it in the utils. Might want to fix the `view!` macro implementation outside this PR.

Reply:
Let me take a look at that.